api/tests/Archive/PostArchiveApiTest.php

```diff
@@ -45,12 +45,12 @@ public function testPostArchiveWithExpirationOK(): void
         $filePrefix = $this->getTestDataSetDir();
         $files = [
             [
-                'uri' => $filePrefix.'/three.jpg',
-                'path' => 'one/two/three.jpg',
+                'uri' => $filePrefix.'/seven.jpg',
+                'path' => 'one/two/seven.jpg',
             ],
             [
-                'uri' => $filePrefix.'/four.txt',
-                'path' => 'four.txt',
+                'uri' => $filePrefix.'/eight.txt',
+                'path' => 'eight.txt',
```

Comment:
Why this changed?

Reply:
because I have SQLSTATE[23000]: Integrity constraint violation: 19 UNIQUE constraint failed: archive.identifier

Reply:
OK :+1: